packages/firestore/src/local/shared_client_state.ts

```diff
@@ -800,3 +1001,11 @@ export class MemorySharedClientState implements SharedClientState {
 
   shutdown(): void {}
 }
+
+const EMPTY_NUMBERED_ID_SET = new SortedSet<number>(primitiveComparator);
+function targetIdSet(): SortedSet<TargetId> {
+  return EMPTY_NUMBERED_ID_SET;
+}
+function batchIdSet(): SortedSet<BatchId> {
+  return EMPTY_NUMBERED_ID_SET;
+}
```

Comment:
Move these to collections.ts as EMPTY_BATCH_ID_SET and EMPTY_TARGET_ID_SET ?  (and I probably wouldn't bother sharing an underlying EMPTY_NUMBERED_ID_SET as I don't think we're that desperate to save on object count).

Reply:
That works for me. Done.